app/Http/PsrRouterServiceProvider.php

```diff
@@ -580,4 +583,9 @@ private function bindUiTPASOrganizers(Router $router): void
             $routeGroup->get('{organizerId}/card-systems/', GetCardSystemsFromOrganizerRequestHandler::class);
         });
     }
+
+    private function bindMailinglist(Router $router): void
+    {
+        $router->put('mailinglist/{email}/{mailingListId}/', SubscribeUserToMailinglistRequestHandler::class);
```

Comment:
The order of the parts of the URL path is a bit weird, `mailinglist/{mailingListId}/{email}` seems more logical.
Even better would be `PUT mailinglist/{mailingListId}` and the body contains the email.

If the front end is the only consumer, it could be worth changing?